### DISCUSSION TITLE
Erro Criação Migrate prisma migrate dev

### DISCUSSION THREAD

**Discussion by ftarrone** [prisma/prisma#9860] [Q&A]
Estou iniciando, com a utilização do Prisma, realizar a instalação do CLI e as dependências, mas sempre que tento gerar uma migrate, me deparo com esse erro, ele chega a gerar a migrate

$ yarn prisma migrate dev
yarn run v1.22.17
$ C:\nlw\nodeHeat\node_modules\.bin\prisma migrate dev
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": SQLite database "dev.db" at "file:./dev.db"

SQLite database dev.db created at file:./dev.db

Applying migration `20211020021322_`

The following migration(s) have been created and applied from new schema changes:

migrations/
  └─ 20211020021322_/
    └─ migration.sql

Your database is now in sync with your schema.

Running generate... (Use --skip-generate to skip the generators)
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
error An unexpected error occurred: "EPERM: operation not permitted, unlink 'C:\\nlw\\nodeHeat\\node_modules\\prisma\\query_engine-windows.dll.node'".
info If you think this is a bug, please open a bug report with the information provided in "C:\\nlw\\nodeHeat\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
Error: Command failed with exit code 1: yarn add -D prisma@3.3.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Fernando@NandoPC MINGW64 /c/nlw/nodeHeat
$
o com esse erro. s

**Comment by ryands17** [comment]
@ftarrone 👋 

This seems like a bug. It would be great if you could open one [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with your schema, reproduction steps, and Prisma version so that we can look into this 🙂

**Comment by aqrln** [comment]
Hi @ftarrone! This seems to be a known issue which we are working on, please see here for details: https://github.com/prisma/prisma/issues/9184

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
